### ISSUE TITLE
Installing CoE without Office groups? Not wanting to communicate, only get inventory.

### ISSUE THREAD

**Issue by HenningStrand** [microsoft/coe-starter-kit#5604]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

I want to install core only to get inventory, but not to trigger any communication to makers or users. Documentation states that there should be created three O365 groups, which I hope is OK to not do. While running the setup wizard I see that I can bypass the Configure communication step without providing groups, so I suppose so. However when I get to the Run inventory flows step it seems like the 'Admin | Add Maker to Group' flow is required. I again hope it's OK to leave this off to proceed. 

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.4.2

### What app or flow are you having the issue with?

Setup Wizard

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
Yes you can do that. You'll end up with the inventory this way and all the makers will be listed in our "Maker" table in the kit. So if you ever change your mind you can backfill that list into an m365 group. 

Note also if you are wanting to generally speaking not communicate anything with the makers as you continue (ex looking at governance) you can use the **_Production Environment_** env var. When set to **_no_**, it sends all kit communication to the admins instead. 

![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/a1e33d6b-bab4-4ce0-9461-71398213ca27)
